cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -80,9 +80,11 @@ class CallTest extends DocumentingTest {
 
     section("View the signature for a procedure", "call-view-the-signature-for-a-procedure") {
       p("""To `CALL` a procedure, its input parameters need to be known, and to use `YIELD`, its output parameters need to be known.
-          #The built-in procedure `dbms.procedures` returns the name, signature and description for all procedures.
+          #The `SHOW PROCEDURES` command returns the name, signature and description for all procedures.
```

Comment:
```suggestion
          #The `SHOW PROCEDURES` command can return the name, signature and description for all procedures.
```
fits better with the next part?

Reply:
Or something about it only being parts of the returned values, since the next sentence adds two other columns

Reply:
Yes, nice suggestion :)

Reply:
Or something about it only being parts of the returned values, since the next sentence adds two other columns

Reply:
Or something about it only being parts of the returned values, since the next sentence adds two other columns